.github/workflows/crux-llvm-build.yml

```diff
@@ -95,7 +95,7 @@ jobs:
           case ${{ matrix.ghc }} in
             9.0.2) GHC_NIXPKGS=github:nixos/nixpkgs/nixos-22.05 ;;
             9.2.2) GHC_NIXPKGS=github:nixos/nixpkgs/nixos-22.05 ;;
```

Comment:
Any reason to still keep this line around?

Reply:
Just a little bit of hoarding instinct: while 9.2 is "in flux", having those historic touchpoints makes me feel safer.   I tend to reference CI build processes of various projects as a fairly definitive and explicit way of building things, and so this might be useful to anyone with similar habits that happens to be trying to use 9.2.2 instead of 9.2.4.

That said, there's always git history (although archaeology is not always the easiest way to find information), so there's no really good reason to keep this if you have a strong preference for its removal.

Reply:
OK. I don't feel too strongly about keeping around the configurations for old minor releases—we'll just have to remember to update things like old freeze files.